src/Sentry/SentryOptions.cs

```diff
@@ -527,6 +521,12 @@ public SentryOptions()
                 () => ExceptionProcessors ?? Enumerable.Empty<ISentryEventExceptionProcessor>()
             };
 
+            // from 3.0.0 uses Enhanced (Ben.Demystifier) by default which is a breaking change
+            // unless you are using .NET Native which isn't compatible with Ben.Demystifier.
+            StackTraceMode = Runtime.Current.Name == ".NET Native"
+                ? StackTraceMode.Original
+                : StackTraceMode.Enhanced;
+
```

Comment:
Why did we need to move it?

Reply:
To leave it next to 
```csharp
            SentryStackTraceFactory = new SentryStackTraceFactory(this);

            ISentryStackTraceFactory SentryStackTraceFactoryAccessor() => SentryStackTraceFactory;
```
but I'll undo it since the change doesn`t fix the problem.

Reply:
Just curious if it serves a specific purpose. It wasn't obvious from the diffs 😉